jdaviz/configs/cubeviz/plugins/slice/slice.py

```diff
@@ -9,14 +9,33 @@
 from jdaviz.core.events import AddDataMessage, SliceToolStateMessage, SliceSelectSliceMessage
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import PluginTemplateMixin
+from jdaviz.core.user_api import PluginUserApi
 
 __all__ = ['Slice']
 
 
 @tray_registry('cubeviz-slice', label="Slice")
 class Slice(PluginTemplateMixin):
+    """
+    See the :ref:`Slice Plugin Documentation <slice>` for more details.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``slice``
+      Current slice number.
+    * ``wavelength``
+      Wavelength of the current slice.  If setting, will update automatically to the nearest
+      slice.
```

Comment:
```suggestion
      Wavelength of the current slice.  When setting this value directly, it will update automatically
      to the value at the nearest integer slice index.
```
If we keep the `setting_` prefix, this phrasing could cause some confusion (is "setting" here talking about the same thing as in the attribute names?). My suggestion might be a little overly wordy/precise, but might be clearer.

Reply:
New wording sounds good to me.